index.Rmd

```diff
@@ -31,7 +31,7 @@ sm <- raster("dsa2_2m.grd")
 sm_crop <- crop(sm, extent(615000, 618000, 232000, 234000))
 ```
 
-4. take a look at empanada
+4. take a look at empanada 
```

Comment:
Nice plot! it would be better if you could add a title, x-axis label, y-axis label, and legend in the plot.